starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <title>Apple Pie Recipe</title>
+
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title> Apple Pies</title>
+  <link rel="stylesheet" typle="type/text" href="css/reset.css">
+  <link rel="stylesheet" typle="type/text" href="css/main.css">
 </head>
+
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
-
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
-
-Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
-lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-that it does not run off.
-
-Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
+  <section class="hero-section">
+    <h1>Apple Pie</h1>
+  </section>
+  <div class="container">
+    <p class="intro">
+      This was my grandmother's apple pie recipe. I have never seen another one <br>quite like it. It will always be my favorite and has won me several first place <br>prizes in local competitions. I hope it becomes one of your favorites as well!
+    </p>
+
+    <img class="recipe" src="/Users/randyarbolaez/code/Week1/Monday/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/recipe-info.png"
```

Comment:
Make sure you start from where you are now and then "navigate" to the destination folder. This being said:
```
<img class="recipe" src="./images/recipe-info.png">
```